index.html

```diff
@@ -177,10 +178,12 @@ <h2 id="cont">Contact me</h2>
         <p class="textab" id="conttext">If you have an application you are intrested in developing, a feature that you need built or 
           a project that needs coding. I'd love to help with it</p>
       </article>
-      <input class="input" type="text" value="John">
-      <input class="input" type="email" value="john@gmail.com">
-      <textarea class="input" cols="30" rows="10">Write your message here</textarea>
-      <a id="getIt" href="" class="prLnk">Get It Touch</a>
+      <form action="https://formspree.io/f/xnqlwozn" method="POST">
+        <input class="input" type="text" placeholder="John" size="30" required>
```

Comment:
- Kindly work on this change:
![Screen Shot 2021-09-21 at 11 30 37](https://user-images.githubusercontent.com/22258274/134210260-35320660-a4b1-49ea-a38c-11a06eadaa2f.png)